### DISCUSSION TITLE
Card: G20

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#2] [Ideas]
- Clarification: was "your first played op" meant to mean an Ops during the players *next* AP, and only within the current turn?
- Yes. Basically, the first card played after the header phase, if played for ops.
- That's the reverse understanding of what I wrote as an assumption. So we want something like this?
    > All further cards played by the player this turn have their Ops value increased by 1.
    > **Four players partaking**: Player’s Ops is increased by 1 more in the first Action Phase after the Headline Phase.

**Comment by rodrigoSantamaria** [comment]
Exactly, that would be a right phrasing of what the card is intended to do

**Comment by RebelusQuo** [comment]
Acknowledged
